libs/data-access/api/src/lib/trpc/routers/course.router.ts

```diff
@@ -156,6 +161,11 @@ export const courseRouter = t.router({
 					code: "FORBIDDEN",
 					message: "Editing a course requires either: admin role | author of course"
 				});
+			} else if (input.course.authors.length <= 0 && ctx.user.role != "ADMIN") {
+				throw new TRPCError({
+					code: "FORBIDDEN",
+					message: "Deleting the last author as is not allowed, except for Admin Users. Contact the side administrator for more information. "
```

Comment:
die message ist doppelt. Da aber bald translations kommen, ist das hier nicht so wild. In anderen fällen sollte das aber vermieden werden